scripts/poe/lib/utils.js

```diff
@@ -10,17 +10,25 @@ module.exports = {
   error: (msg) => {
     console.log(`${chalk.red('Error: ')}${msg}`);
   },
-  extractPlaceholders: (file) => {
+  extractPlaceholders: (file, onlyPlaceholders = false) => {
     const content = fs.readFileSync(file, 'utf8');
 
+    if (onlyPlaceholders) {
+      return content
+              .toString()
```

Comment:
Because you've specified an encoding, `content` is a string so you don't need to `toString`.